packages/api-request-builder/package.json

```diff
@@ -3,6 +3,10 @@
   "publishConfig": {
     "access": "public"
   },
+  "engines": {
+    "node": ">=14",
+    "npm": ">=6.0.0"
```

Comment:
Nit: in each package, I would omit the `npm` part and only keep `node`.

Do you have a reason for keeping both?

Reply:
Not really, now that we have included it within the resolution section of in the `package.json` I think this can as well go.